recipe/meta.yaml

```diff
@@ -39,25 +39,24 @@ requirements:
   host:
     - python
     - pip
-    - numpy
     - setuptools
     - wheel
   run:
     - {{ compiler('cxx') }}  # [not win]
     - clang                  # [win]
     - clangxx                # [win]
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy
     - gast >=0.5.0,<0.6.0
     - ply >=3.4
     - beniget >=0.4.0,<0.5.0
-    - xsimd 11.0.0
+    - xsimd 12.1.1
```

Comment:
Should we add setuptools considering it's needed to use `pythran.dist` and the fact that they specify it in their dependencies (https://github.com/serge-sans-paille/pythran/blob/master/requirements.txt#L2)?

Reply:
yes, done. thank you